spec/features/merchants/invoices/show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe 'Merchant Invoices show page' do
+  before :each do
+    @merchant_1 = Merchant.create!(name: 'Primate Privleges')
+
+    @item_1 = @merchant_1.items.create!(name: 'Monkey Paw', description: 'A furry mystery', unit_price: 1000)
+    @item_2 = @merchant_1.items.create!(name: 'Gorilla Grip Glue', description: 'A sticky mystery', unit_price: 2000)
+
+    @customer_1 = Customer.create(first_name: 'Teddy', last_name: 'Wagner')
+
+    @invoice_1 = @customer_1.invoices.create!(status: 1)
+    @invoice_2 = @customer_1.invoices.create!(status: 1)
+
+    @invoice_item_1 = @invoice_1.invoice_items.create(item_id: @item_1.id, quantity: 3, unit_price: 1000, status: 1)
+    @invoice_item_2 = @invoice_2.invoice_items.create(item_id: @item_2.id, quantity: 1, unit_price: 2000, status: 1)
+
+
+
+    visit "/merchants/#{@merchant_1.id}/invoices/#{@invoice_1.id}"
```

Comment:
you could play around with the advanced routing i.e. merchant_path in place of the URI --- this one may be tricky though so a different one may be a better first go at it.